releasenotes/notes/admm-optimizer-d482b33f2474244e.yaml

```diff
@@ -4,3 +4,8 @@ features:
     Removed soft dependency on CPLEX in ADMMOptimizer. Now default optimizers used by ADMMOptimizer
     are MinimumEigenOptimizer for QUBO problems and SlsqpOptimizer as a continuous optimizer. You
     can still use CplexOptimizer as an optimizer for ADMMOptimizer, but it should be set explicitly.
+  - |
```

Comment:
We should be using a new reno for each PR - otherwise things will be mixed up and complicated. So can you please make a new reno with the new text. Thx.

Reply:
OK. Done.

Reply:
I made a minor tweak to spelling. Since these form the user facing release notes it saves having to edit them later when they are checked at release time and any spelling errors get fixed.

Reply:
Thanks a lot for fixing spelling!